caas/kubernetes/provider/k8s_test.go

```diff
@@ -339,6 +341,21 @@ type K8sBrokerSuite struct {
 
 var _ = gc.Suite(&K8sBrokerSuite{})
 
+func (s *K8sBrokerSuite) TestAPIVersion(c *gc.C) {
+	ctrl := s.setupBroker(c)
+	defer ctrl.Finish()
+
+	r := rest.NewRequest(nil, "get", &url.URL{Path: "/path/"}, "", rest.ContentConfig{}, rest.Serializers{}, nil, nil, 0)
+	gomock.InOrder(
```

Comment:
Do you need the InOrder here? There's only one expectation.

Reply:
guess not